Code/Yahoo.Yui.Compressor.Tests/TestHelpers/BuildEngine.cs

```diff
@@ -16,31 +18,29 @@ public static bool DoesNotContainError(IBuildEngine engine, string error)
             return CheckForError(engine, error, false);
         }
 
-        private static bool CheckForError(IBuildEngine engine, string error, bool exists)
+        private static bool CheckForError(IBuildEngine engine, string searchString, bool exists)

```

Comment:
My previous comment about this method still applies.

> Since the method is now checking for Errors OR Messages, then CheckForError is no longer an appropriate name for the method - CheckForMessage probably makes more sense since a message could be an error or a regular message.
Also ideally the messageType would be an enum since there are only two valid values.

But since you no longer test the output message then an alternative fix is to just revert you change completely - and leave it as a "CheckForError" method.
Please revert your change or make the changes I suggested